ci-operator/jobs/openshift/release/openshift-release-release-4.3-periodics.yaml

```diff
@@ -6910,6 +6910,8 @@ periodics:
         value: "4.3"
       - name: CLUSTER_TYPE
         value: openstack-ppc64le
+      - name: SSHD_43_BASTION_PPC64LE_IP
```

Comment:
Variable for this makes sense, so you can tune per job.  Why `SSHD_43_BASTION_PPC64LE_IP` instead of `SSHD_BASTION_IP`?  Or `HOST_ALIAS_IP`, since that's where it's getting plugged into the template?

Reply:
SSHD_BASTION_IP does sound more reasonable (and generic for 4.4 & 4.5) now that you mention it.